captcha-plugin-draft.txt

```diff
@@ -107,6 +107,13 @@ We provisionally instantiate these with
 
 [TODO: I would really like to use something other than RSA, but haven't found
 satisfactory research on alternative blind signature schemes.]
+[Alex: It could be possible to use a DSA-style blind signature (see
+http://blog.cryptographyengineering.com/p/note-on-blind-signature-schemes.html).
+The protocol requires three steps, the additional step is required since the
+server initially sends a random element for each signature that needs to be
+blindly signed. While this is not ideal it may be possible to incorporate
+this into the structure below so that the plugin asks for 10000 elements of
+randomness from the server when it generates the nonces.]
```

Comment:
When we talked to @burdges about this, he said Tanja Lange advised them to just use RSA blinding. I'm open to other other options, but we'll need to run it by some real cryptographers.